Update docstring of the parameter sortby in MapSequence Class. (#3601)
sunpy/sunpy: changelog/3601.doc.rst

> Updated the docstring for the parameter sortby in MapSequence class. Added information about Valid Sorting Strategies, Default Sorting Strategies and disabling sorting.

Reviewer: ```suggestion
Updated the docstring for the parameter ``sortby`` in `~sunpy.map.MapSequence` with the default value, valid value and how to disable sorting.
```
